### PR TITLE
Fixes saving private note about customer from order page

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Fixes described issue.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/13428
| How to test?  | Saving private note from Order page should work now.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13916)

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php [hunk 1]
```diff
@@ -279,6 +279,13 @@ public function savePrivateNoteAction($customerId, Request $request)
                     $data['note']
                 ));
 
+                if ($request->isXmlHttpRequest()) {
```
Comment (sarjon) [inline src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php:282]: @matks this is something we could improve. When we need to perform "Save private note" action using both form submit (with page refresh and flash message) and ajax request, I think there should be a better way to handle it.

**Comment by sarjon** [comment]
@matks this should be part of 1.7.6.

File: src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php (hunk 1 above)
Comment (matks) [reply src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php:282]: How would you improve it ? Maybe 2 controller actions, in order to separate clearly the 2 usecases, as the input payload is very different and the output response is very different too

Comment (sarjon) [reply src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php:282]: Well, this should clearly be part of BO Http API. But I don't know if we can put it under `Controller/BackOffice/Api` or similar.

**Comment by sarjon** [comment]
Ready for review.

**Review by matks — APPROVED** [review]

**Comment by sarjon** [comment]
@matks Ship it. :ship:

**Comment by matks** [comment]
> @matks Ship it. 🚢

I would like to say


But PrettyCI is trolling us

**Comment by sarjon** [comment]
Can you rerun Pretty CI? :running_man:

**Comment by matks** [comment]
> Can you rerun Pretty CI? 🏃

It is lost in the queue. I'm going to bypass it I think

**Comment by matks** [comment]
@Quetzacoalt91 re-runned it and it worked 😢I believe PrettyCI is trolling *me*

**Pull request merged**